### PR TITLE
Adds emote to crack your knuckles.

### PR DESCRIPTION
**Pull request by mrhugo13**
## About The Pull Request
Added a new sound file named `knuckles.ogg`, and added an emote which lets you crack your knuckles together.

https://streamable.com/yjor8

## Why It's Good For The Game
It would be cool.

## Changelog
:cl:Vondiech
add: You can now crack your knuckles with *crack
/:cl:

### CODE REVIEW

**Review by Rohesie — DISMISSED** [review]
Code-wise is fine. Simple PR.

**Comment by HomocidlePancakes** [comment]
*cracks knuckles*
Omae wa mou shinderu.

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -34,6 +34,12 @@
 	message = "crosses their arms."
 	restraint_check = TRUE
 
+/datum/emote/living/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles together."
```
Comment (ATH1909) [inline code/modules/mob/living/emote.dm]: what if you only have one hand

this should be "cracks their knuckles." instead

Comment (mrhugo13) [reply code/modules/mob/living/emote.dm]: Fixed it with your suggestion.

**Comment by mrhugo13** [comment]
Re-requesting review for the sake of checking if any of my code is dumb or considered bad practice.

File: code/modules/mob/living/carbon/emote.dm [hunk 2]
```diff
@@ -34,6 +34,17 @@
...
 							'sound/misc/clap4.ogg')
 
+/datum/emote/living/carbon/crack
+	key = "crack"
+	key_third_person = "cracks"
+	message = "cracks their knuckles."
+	sound = 'sound/misc/knuckles.ogg'
+
+/datum/emote/living/carbon/crack/can_run_emote(mob/living/carbon/user, status_check = TRUE , intentional)
+	. = ..()
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/emote.dm]: `return ..()` at the end of the proc instead

**Comment by Anonmare** [comment]
very fast knucklcracking at very hihg speeds

File: code/modules/mob/living/carbon/emote.dm (hunk 2 above)
Comment (mrhugo13) [reply code/modules/mob/living/carbon/emote.dm]: as you please

**Comment by necromanceranne** [comment]
Get GotNS and then set the warcry to *crack.

Arthritis of the North Star.

**Pull request merged**